DataModel/Migrations/CctDbContextModelSnapshot.cs

```diff
@@ -15,7 +14,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "3.1.9")
+                .HasAnnotation("ProductVersion", "3.1.8")
```

Comment:
Should this be product version 3.1.9?

Reply:
Hi Michelle, Yes, please do not merge with Master. I just had a quick meeting with Alfred